pnpm-lock.yaml

```diff
@@ -163,6 +163,9 @@ importers:
       '@react-aria/utils':
         specifier: ^3.23.0
         version: 3.23.0(react@18.2.0)
```

Comment:
Lagt til `@types/node` for at TS skal kompilere ved bruk av `process.env.NODE_ENV`